_includes/footer.html

```diff
@@ -12,7 +12,7 @@
           </div>
           <ul class="footer-nav-list collapse" id="footerDocList">
             <li><p><a href="{{ site.baseurl }}/docs/introduction">Introduction</a></p></li>
-            <li><p><a href="{{ site.baseurl }}/docs/quickstart-java.html">Quick Start</a></p></li>
+            <li><p><a href="{{ site.baseurl }}/docs/quickstart/">Quick Start</a></p></li>
```

Comment:
Let's have `quick-start` instead of the `quickstart` to be according to the folder name.